benchmark/single-source/NSStringConversion.swift

```diff
@@ -14,16 +14,93 @@
 import TestsUtils
 import Foundation
 
-public let NSStringConversion = BenchmarkInfo(
-  name: "NSStringConversion",
-  runFunction: run_NSStringConversion,
-  tags: [.validation, .api, .String, .bridging])
+public let NSStringConversion = [
+  BenchmarkInfo(name: "NSStringConversion",
+                runFunction: run_NSStringConversion,
+                tags: [.validation, .api, .String, .bridging]),
+  BenchmarkInfo(name: "NSStringConversion.UTF8",
+                runFunction: run_NSStringConversion_nonASCII,
+                tags: [.validation, .api, .String, .bridging],
+                setUpFunction: setUpNSStringConversion_nonASCII),
```

Comment:
I suspect this would be a lot more readable if got rid of the run/setup functions and used closures instead. (It's a shame BenchmarkInfo is so unpleasant to initialize; I believe we have plans to address that at some point.)

Also, since all of these require the Objective-C runtime, we could just define the benchmark list empty on non-Darwin platforms:

```swift
#if !_runtime(_ObjC)
public let NSStringConversion = []
#else 
public let NSStringConversion = [
  BenchmarkInfo(...),
  ...
]
#endif 
```

Reply:
ok